EIPS/eip-XXXX.md

```diff
@@ -0,0 +1,42 @@
+---
+eip: TBD
+title: Point Evaluation Precompile Gas Cost Increase
+description: Increases cost of point evaluation precompile
+author: Marcin Sobczak (@marcindsobczak), Kamil Chodoła (@kamilchodola), Marek Moraczyński (@MarekM25)
+discussions-to: TBD
```

Comment:
Please create a discussions topic in Eth Magicians using the template: https://ethereum-magicians.org/c/eips/5